src/main/java/org/opentripplanner/netex/loader/NetexXmlParser.java

```diff
@@ -23,6 +24,15 @@
     PublicationDeliveryStructure parseXmlDoc(byte[] bytesArray) throws JAXBException {
         JAXBElement<PublicationDeliveryStructure> root;
         ByteArrayInputStream stream = new ByteArrayInputStream(bytesArray);
+        return parseXmlDoc(stream);
+    }
```

Comment:
This method is no longer in use and should be deleted.

Reply:
I wasn't sure if it was needed somewhere else -- it's been removed.